src/augtool.c

```diff
@@ -426,6 +427,28 @@ static void print_aug_error(void) {
     }
 }
 
+static void sigint_handler(int unused)
+{
+    (void)unused;
+
+    // Cancel the current line of input, along with undo info for that line.
+    rl_replace_line("", 1);
+
+    // Move the cursor to the next screen line, then force a re-display.
+    rl_crlf();
+    rl_forced_update_display();
+}
+
+static void install_signal_handlers(void)
+{
```

Comment:
It's a really minor nit, but the Augeas code base has opening braces on the same line as the function prototype